tests/unit/components/SharingOptionsLink.spec.js

```diff
@@ -176,7 +176,7 @@ describe('SharingOptionsLink', () => {
     // Close the popup
     $popup.instance.closed = true
     // Wait for the interval to be called
-    jest.advanceTimersByTime(1000)
+    vi.advanceTimersByTime(1000)
```

Comment:
good to see that vitest supports this kind of feature

Reply:
Vitest was made so "migrating from Jest is straightforward" and I can testify it is true!